anchore_engine/db/entities/upgrade.py

```diff
@@ -1183,6 +1183,245 @@ def db_upgrade_012_013():
+        ('image_gems', 'source_pkg'),
+        ('image_gems', 'latest'),
+
+        # ImageCpe
+        ('image_cpes', 'image_user_id'),
+        ('image_cpes', 'image_id'),
+        ('image_cpes', 'pkg_type'),
+        ('image_cpes', 'pkg_path'),
+        ('image_cpes', 'cpetype'),
+        ('image_cpes', 'vendor'),
+        ('image_cpes', 'name'),
+        ('image_cpes', 'version'),
+        ('image_cpes', 'update'),
+        ('image_cpes', 'meta'),
+
+        # FilesystemAnalysis
+        ('image_fs_analysis_dump', 'image_user_id'),
+        ('image_fs_analysis_dump', 'image_id'),
+        ('image_fs_analysis_dump', 'compressed_content_hash'),
+        ('image_fs_analysis_dump', 'compression_algorithm'),
+
+        # AnalysisArtifact
+        ('image_analysis_artifacts', 'image_id'),
+        ('image_analysis_artifacts', 'image_user_id'),
+        ('image_analysis_artifacts', 'analyzer_id'),
+        ('image_analysis_artifacts', 'analyzer_artifact'),
+        ('image_analysis_artifacts', 'analyzer_type'),
+        ('image_analysis_artifacts', 'artifact_key'),
+
+        # Image
+        ('images', 'id'),
+        ('images', 'user_id'),
+        ('images', 'digest'),
+        ('images', 'distro_name'),
+        ('images', 'distro_version'),
+        ('images', 'like_distro'),
+        ('images', 'dockerfile_mode'),
+
+        # ImagePackageVulnerability
+        ('image_package_vulnerabilities', 'pkg_user_id'),
+        ('image_package_vulnerabilities', 'pkg_image_id'),
+        ('image_package_vulnerabilities', 'pkg_name'),
+        ('image_package_vulnerabilities', 'pkg_version'),
+        ('image_package_vulnerabilities', 'pkg_type'),
+        ('image_package_vulnerabilities', 'pkg_arch'),
+        ('image_package_vulnerabilities', 'pkg_path'),
+        ('image_package_vulnerabilities', 'vulnerability_id'),
+        ('image_package_vulnerabilities', 'vulnerability_namespace_name'),
+
+        # DistroMapping
+        ('distro_mappings', 'from_distro'),
+        ('distro_mappings', 'to_distro'),
+        ('distro_mappings', 'flavor'),
+
+
+        # PolicyEvaluationCache
+        ('policy_engine_evaluation_cache', 'user_id'),
+        ('policy_engine_evaluation_cache', 'image_id'),
+        ('policy_engine_evaluation_cache', 'eval_tag'),
+        ('policy_engine_evaluation_cache', 'bundle_id'),
+        ('policy_engine_evaluation_cache', 'bundle_digest'),
+        ('policy_engine_evaluation_cache', 'user_id')
+    ]
+
+    log.err('Updating tables')
+    for table_name, column_name in to_update:
+        log.err('Updating table {} column {} to varchar'.format(table_name, column_name))
+        try:
+            rc = engine.execute('ALTER TABLE {} ALTER COLUMN "{}" type varchar'.format(table_name, column_name))
+        except Exception as ex:
+            log.err('Failed updating {} column {} type to varchar. Err: {}'.format(table_name, column_name, str(ex)))
+            raise ex
+
+
+def db_upgrade_013_014():
```

Comment:
This will conflict w/ the global max images / additional archival rule PR

Reply:
Yes, I've resolved it in the rebase operation to move your code to a new function name and call that function from this function.